chingu-voyages/v51-tier3-team-33
# Feature/create homepage

# Description
Created responsive UI for homepage view


## Main changes explained:
- Added components in app/home/ui for responsive design of `/home` page 

## How to test:
1. check into current branch
2. do `pnpm install` and `pnpm run dev` to run this PR locally
3. log into app using google credentials
4. go to /home
5. verify the home page design 
6. verify that your username and email are displayed in the homepage
7. verify that mobile view of the page displays normally

## Screenshots or videos of changes:

![Screenshot 2024-09-20 at 3 37 11 PM](https://github.com/user-attachments/assets/07a0bbd7-65a1-4945-b7d2-d7c5be57ff6c)


## Note:
This is just UI, ether is no functionality yet.
